accelerate.cabal

```diff
@@ -505,22 +487,38 @@ library
   hs-source-dirs:
         src
 
-  if flag(debug) || flag(ekg)
-    ghc-options:
-        -optc-DACCELERATE_DEBUG
+  if flag(debug) || flag(prof)
+    cc-options:
+        -DACCELERATE_DEBUG
 
     cpp-options:
         -DACCELERATE_DEBUG
 
-  if flag(ekg)
+  if flag(prof)
+    cc-options:
+        -DACCELERATE_PROFILE
+
+    cxx-options:
+        -O3
+        -Wall
+        -march=native
```

Comment:
Because we need to compile C[++] code ourselves with TH rather than via Cabal, I think it gets picked up with `ghc-options -optcxx=...` setting, but either way if you `stack build --ghc-options=-v` you'll see them come up.